src/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+    integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+  <title>To-do list</title>
+</head>
+
+<body>
+  <main class="main">
```

Comment:
- [ ] It would be better if we achieve the given design. Here is how our project looks compared to the given design
Our Project
![Screenshot from 2021-09-23 03-30-37](https://user-images.githubusercontent.com/46388113/134451519-0ec3d8f6-5e49-418f-a2f7-8d65341f4b53.png)
Given DesigN
![Screenshot from 2021-09-23 03-41-30](https://user-images.githubusercontent.com/46388113/134451598-d546e839-1d51-47a2-8634-02e99c2ab314.png)

Reply:
Hi, I think you opened the index.html from src, which is why it looks like that. If you open the one from dist, it should look like this:

<img width="1440" alt="Screen Shot 2021-09-23 at 12 44 40 PM" src="https://user-images.githubusercontent.com/62937819/134456059-8e9b6304-5edf-4d61-aa90-89f9b98abbe1.png">

Reply:
Yes, sure @nikoescobal. I did `npm i` but I still couldn't find the `dist` directory, so I had to use the `src/index.html` file. Great job!!! :fireworks: